docs/CODE_OF_CONDUCT-ko.md

```diff
@@ -18,15 +18,15 @@
 * 부적절한 것으로 간주될 수 있는 다른 행위
 
 프로젝트 유지자는 이 행동 강령을 따르지 않은 댓글, 커밋, 코드, 위키 편집, 이슈와 그 외 다른 기여를
-삭제, 수정 또는 거부할 권리와 책임이 있습니다. 또한, 부적당하거나 험악하거나 공격적이거나 해롭다고
+삭제, 수정 또는 거부할 권리와 책임이 있습니다. 또한, 부합하지 않거나 험악하거나 공격적이거나 해롭다고
```

Comment:
```suggestion
삭제, 수정 또는 거부할 권리와 책임이 있습니다. 또한, 행동 강령 규약에 부합하지 않거나 험악하거나 공격적이거나 해롭다고
```

Reply:
hmmm I kinda think this sounds too aggressive (just my thoughts)

Reply:
@Helloyunho I think it’s adequate action for malicious contributors.